java/sage/MediaServer.java

```diff
@@ -560,6 +594,10 @@ protected void sizeFile() throws java.io.IOException
           availSize = xcoder.getVirtualTranscodeSize();
           totalSize = xcoder.isTranscodeDone() ? availSize : getLargeFileSize(currFile.toString());
         }
+        if (remuxer != null)
+        {
+          availSize = totalSize = remuxer.getFileSize();
```

Comment:
You'll want to use getLargeFileSize(String) for the totalSize if the file is recording. This is used as an indicator in some of the playback clients to know that the file is currently recording (they do it by checking if totalSize > availSize); also be sure you handled the circular file case as that will be used for the preview window during playback (although your remuxer logic may already handle that case, I haven't got that far reviewing the code yet :) )